packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+  * Slider label value.
+  */
+  labelName: PropTypes.string.isRequired,
```

Comment:
Should this prop be named something like labelText?  Don't we have that same name for some other Terra components?

Reply:
I am sure I saw labelName somewhere and decided to use that. But I did see labelText as well in terra-core and other repos. So, I will update this.

Reply:
Updated to labelText.